.chloggen/fix-clickhouse-create-database.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: clickhouseexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix no such host when create database
```

```suggestion
note: Fix "no such host" error when the DSN hostname contains the database name
```
Wording suggestion just to clarify the scope of the fix for users.